### ISSUE TITLE
Check failed: status == CUDNN_STATUS_SUCCESS (7 vs. 0)Failed to set cuDNN stream.

### ISSUE THREAD

**Issue by alaa-Mz** [tensorflow/tensorflow#25709]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
- OS Platform and Distribution: Windows 10 
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: no 
- TensorFlow installed from (source or binary):source 
- TensorFlow version (use command below): 1.12.0
- Python version: 3.6
- Bazel version (if compiling from source):N/A
- GCC/Compiler version (if compiling from source):N/A
- CUDA/cuDNN version: 9.0 / 7.1
- GPU model and memory:  GTX 1050 Ti

**Describe the current behavior**
i tried to train the model by using image datasets

**Code to reproduce the issue**
    
    from keras.callbacks import ModelCheckpoint, EarlyStopping
    from model import load_model
    import numpy as np 
    import argparse
    parser=argparse.ArgumentParser()
    parser.add_argument('n_epochs',type=int)

    args=parser.parse_args()
    X_train=np.load('training.npy')
    frames=X_train.shape[2]
    #Need to make number of frames divisible by 10

    frames=frames-frames%10
    X_train=X_train[:,:,:frames]
    X_train=X_train.reshape(-1,227,227,10)
    X_train=np.expand_dims(X_train,axis=4)
    Y_train=X_train.copy()

    epochs=args.n_epochs
    batch_size=1

    if __name__=="__main__":

	    model=load_model()

	    callback_save = ModelCheckpoint("model.h5",
									monitor="mean_squared_error", save_best_only=True)

	    callback_early_stopping = EarlyStopping(monitor='val_loss', patience=3)

	    print('Model has been loaded')

	    model.fit(X_train,Y_train,
			      batch_size=batch_size,
			      epochs=epochs,
			      callbacks = [callback_save,callback_early_stopping]
			  )

**Output**
2019-02-13 03:45:14.939208: E tensorflow/stream_executor/cuda/cuda_driver.cc:981] failed to synchronize the stop event: CUDA_ERROR_LAUNCH_FAILED: unspecified launch failure

2019-02-13 03:45:14.943572: E tensorflow/stream_executor/cuda/cuda_timer.cc:55] Internal: error destroying CUDA event in context 000001F501169B70: CUDA_ERROR_LAUNCH_FAILED: unspecified launch failure

2019-02-13 03:45:14.948104: E tensorflow/stream_executor/cuda/cuda_timer.cc:60] Internal: error destroying CUDA event in context 000001F501169B70: CUDA_ERROR_LAUNCH_FAILED: unspecified launch failure

2019-02-13 03:45:14.952242: F tensorflow/stream_executor/cuda/cuda_dnn.cc:231] Check failed: status == CUDNN_STATUS_SUCCESS (7 vs. 0)Failed to set cuDNN stream.

**Comment by jvishnuvardhan** [comment]
This is not Build/Installation or Bug/Performance issue. Please post this kind of support questions at [Stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow). There is a big community to support and learn from your questions. GitHub is mainly for addressing bugs in installation and performance. Thanks!
